refactor_me.py

```diff
@@ -2,25 +2,49 @@
 
 from collections import namedtuple, defaultdict
 from operator import itemgetter
+from sys import stdout
+import io
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
 def sum_expenses(expenses, min_amount=0):
+    """Aggregates all Expense entries into a categorized list of sums
+
+    Args:
+        expenses (iter(Expense)): Iterable of Expenses to be summed
+        min_amount (int): Minimum value to match to be included with the result
+
+    Returns:
+        dict: Category-grouped results of summed values
+    """
     aggregated_expenses = defaultdict(int)
-    for expense in expenses:
-        amount = expense.amount
+    for category, amount in expenses:
         if amount >= min_amount:
-            aggregated_expenses[expense.type_] += amount
+            aggregated_expenses[category] += amount
     return aggregated_expenses
 
 
-def print_expenses(expenses):
-    for expense, amount in sorted(expenses.items(), key=itemgetter(1)):
-        print(expense, amount)
+def format_dict(input_dict, write_out=None):
```

Comment:
introducing this function looks like overengineering

Reply:
I agree and don't. We are presented with a method that outputs the content of a dictionary to the terminal.

I first had an issue with it's name as it limited it's purpose. There was step within this method that referenced the Expense objects but instead used all of the methods associated with dictionaries. Second issue was explained in [my comment above](https://github.com/dmugtasimov/refactor-me/pull/8#discussion_r347200719).